### PR TITLE
Default theme reference refactor

### PR DESCRIPTION
**Pull request by Xaekai**
Reduce the number of places the default theme is hardcoded from 6 to 1.

### CODE REVIEW

File: www/js/data.js [hunk 1]
```diff
@@ -97,7 +97,7 @@ function getOrDefault(k, def) {
 }
 
 var USEROPTS = {
-    theme                : getOrDefault("theme", "/css/themes/slate.css"),
+    theme                : getOrDefault("theme", DEFAULT_THEME), // Set in head template
```
Comment (calzoneman) [inline www/js/data.js:100]: Does this actually work?  I thought jade locals were local to the jade renderer.

**Comment by Xaekai** [comment]
The jade local was interpolated into a declaration in a script in the template (line 15 of head.pug). I tested this on PonkTube.

The comments in data and theme are to guide neophyte server operators to the right place to change the default.

The head template seems like the ideal place for that setting to live since it's loaded by every page.

File: templates/head.pug [hunk 2]
```diff
@@ -7,7 +9,10 @@ mixin head()
   title= siteTitle
   link(href="/css/sticky-footer-navbar.css", rel="stylesheet")
   link(href="/css/cytube.css", rel="stylesheet")
-  link(id="usertheme", href="/css/themes/slate.css", rel="stylesheet")
+  link(id="usertheme", href=DEFAULT_THEME, rel="stylesheet")
+
+  script(type="text/javascript").
+    var DEFAULT_THEME = '#{DEFAULT_THEME}';
```
Comment (calzoneman) [inline templates/head.pug:15]: Didn't you mention in your Pug upgrade PR that interpolation is no longer supported?  Or is it still for JS string literals?

Comment (calzoneman) [inline templates/head.pug:15]: I understand that you tested it, I just want to understand what's going on because I'm a little confused.

**Comment by Xaekai** [comment]
Interpolation is only no longer supported inside **element attributes**.

**Comment by calzoneman** [comment]
Oh good, I love non-orthogonal language features

**Pull request merged**